mava/wrappers/gym.py

```diff
@@ -191,6 +195,67 @@ def modify_space(self, space: spaces.Space) -> spaces.Space:
             raise ValueError(f"Space {type(space)} is not currently supported.")
 
 
+class GymToJumanji(gymnasium.Wrapper):
+    """Converts Gym outputs to Jumanji timesteps"""
+
+    def reset(
+        self, seed: Optional[list[int]] = None, options: Optional[list[dict]] = None
+    ) -> TimeStep:
+        obs, info = self.env.reset(seed=seed, options=options)
+
+        num_agents = len(self.env.single_action_space)
+        num_envs = self.env.num_envs
+
+        ep_done = np.zeros(num_envs, dtype=float)
+        rewards = np.zeros((num_envs, num_agents), dtype=float)
+
+        timestep = self._create_timestep(obs, ep_done, rewards, info)
+
+        return timestep
```

Comment:
jumanji has a nice method called `restart` which will do this for you :smile: 

`from jumanji.types import restart`

Reply:
the restart doesn't work in this case since we need the `step_type` to be an array of length `num_envs`
```
return TimeStep(
        step_type=StepType.FIRST,
        reward=jnp.zeros(shape, dtype=float),
        discount=jnp.ones(shape, dtype=float),
        observation=observation,
        extras=extras,
    )
```